helm-chart/values.yaml

```
@@ -222,6 +222,57 @@ cvat:
           protocol: TCP
           name: http
 
+  kvrocks:
+    enabled: true
+    external:
+      host: 127.0.0.1
+    existingSecret: "cvat-kvrocks-secret"
+    secret:
+      create: true
+      name: cvat-kvrocks-secret
+      password: cvat_kvrocks
+    replicas: 1
+    image: apache/kvrocks
+    tag: 2.7.0
+    imagepullploicy: IfNotPresent
+    labels: {}
+    #  test: test
+    annotations: {}
+    # test.io/test: test
+    resources: {}
+    affinity: {}
+    tolerations: []
+    # nodeAffinity:
```

```suggestion
    nodeAffinity: {}
```

Seems like it should be consistent with the other values.